environment.yml

```diff
@@ -0,0 +1,21 @@
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - bump2version
+  - cookiecutter
+  - flake8
+  - pypy3.6
+  - python=3.6
```

Comment:
i'm confused by the 3.6 only here. or is it meant to be an example only?

Reply:
Yes, I'm not sure how to specify a single environment.yml file that would enable all the different possible versions of Python needed to do *all* the testing.  This is just meant as an example to get a first-time contributor up and running.